lib/hider/accounts/user.ex

```diff
@@ -1,13 +1,15 @@
 defmodule Hider.Accounts.User do
   use Ecto.Schema
   import Ecto.Changeset
+  alias Hider.Crypt.{AES, Argon2}
 
   @cast_fields ~w(cpf email first_name last_name middle_name password password_confirmation rg username)a
   @required_fields ~w(cpf email first_name last_name middle_name rg username)a
   @unique_fields ~w(email username rg cpf)a
 
   schema "users" do
     field :cpf, :string
+    field :cpf_blind, :binary
```

Comment:
Add a blind index, a hash, both memory and cpu intensive, to slow down attackers, but saltless (because it's the only way to match then without any novel technique)